src/test/java/org/myrobotlab/codec/CodecUtilsTest.java

```diff
@@ -0,0 +1,48 @@
+package org.myrobotlab.codec;
+
+import org.junit.Test;
+import org.myrobotlab.service.data.Locale;
+import org.myrobotlab.test.AbstractTest;
+
+import static org.junit.Assert.assertEquals;
+
+public class CodecUtilsTest extends AbstractTest {
+
+    @Test
+    public void testLocale() {
+        Locale mrlLocale;
+
+        // String code construction
+        String code = null;
+        String json = null;
+
+        java.util.Locale check = new java.util.Locale("zh-cmn-Hans-CN");
+
+
+        code = "-";
```

Comment:
So "class" is in our json now ?

Reply:
Yes, "class" was chosen since it's an invalid identifier in most languages but it is configurable